RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -655,7 +659,7 @@ void TrackstersMergeProducer::energyRegressionAndID(const std::vector<reco::Calo
   }
 
   // run the inference (7)
-  tensorflow::run(eidSession_, inputList, outputNames, &outputs);
+  tensorflow::run(const_cast<tensorflow::Session *>(eidSession), inputList, outputNames, &outputs);
```

Comment:
`eidSession` is already a `const tensorflow::Session *`: why do you need the `const_cast`?